adapter/grpc/client.go

```diff
@@ -25,25 +25,29 @@ func SentinelUnaryClientIntercept(opts ...Option) grpc.UnaryClientInterceptor {
 			resourceName = options.unaryClientResourceExtract(ctx, method, req, cc)
 		}
 
-		entry, err := sentinel.Entry(
+		entry, blockErr := sentinel.Entry(
 			resourceName,
 			sentinel.WithResourceType(base.ResTypeRPC),
 			sentinel.WithTrafficType(base.Outbound),
 		)
-		if err != nil {
+		if blockErr != nil {
 			if options.unaryClientBlockFallback != nil {
-				return options.unaryClientBlockFallback(ctx, method, req, cc, err)
+				return options.unaryClientBlockFallback(ctx, method, req, cc, blockErr)
 			}
-			return err
+			return blockErr
 		}
 		defer entry.Exit()
 
-		return invoker(ctx, method, req, reply, cc, opts...)
+		err := invoker(ctx, method, req, reply, cc, opts...)
+		if err != nil {
+			sentinel.TraceErrorToEntry(entry, err)
+		}
+		return err
 	}
 }
 
-// SentinelStreamClientIntercept returns new grpc.StreamClientInterceptor instance
-func SentinelStreamClientIntercept(opts ...Option) grpc.StreamClientInterceptor {
+// NewStreamClientInterceptor returns new grpc.StreamClientInterceptor instance
```

Comment:
Maybe we should keep comment consistent with "// NewUnaryClientInterceptor creates the unary client interceptor wrapped with Sentinel entry."

Reply:
Done.